dias/analyzers/dataset_analyzer.py

```diff
@@ -80,7 +80,11 @@ def run(self):
         ds.get.index()
 
         # Use the tracker to get the chimestack files to analyze
-        cs_file_list = self.new_files(filetypes=self.instrument + "_corr")
+        end_time = datetime.datetime.utcnow() - self.offset
```

Comment:
Should we set offset to 0 in the config now?

Reply:
Good idea!

Reply:
```
[2020-11-06 19:46:44,756] dias[dataset]: Now processing acquisition /mnt/gong/staging/20201101T000000Z_chime_flaginput (1 files)
[2020-11-06 19:48:39,835] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00091641_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,009] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00096999_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,124] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00102088_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,284] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00107178_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,312] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00112268_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,319] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00117357_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,327] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00122447_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,332] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00127536_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,339] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00132636_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,345] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00137725_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,352] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00142815_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,359] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00147904_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,365] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00152994_0000.h5, yet. Skipping..
[2020-11-06 19:48:40,372] dias[dataset]: Flags not available for /mnt/gong/staging/20201104T214036Z_chimestack_corr/00158083_0000.h5, yet. Skipping..
```

This is how it will look

Reply:
Should we add an alert/error message if we are skipping data that should already have flags available?

Reply:
As in, data that is more than `24h` old?

Reply:
yes

Reply:
Done!